front_end/helper.py

```diff
@@ -54,8 +54,9 @@ def is_old_file(file_path, days=30):
 def convert_html_to_markdown(text):
```

Comment:
A couple spacing tweaks that bring markdown styling more in line with what an instructor specifies.